WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListActivity.kt

```diff
@@ -335,7 +335,11 @@ class PostsListActivity : LocaleAwareActivity(),
             event.applyIfNotHandled {
                 val fragment = supportFragmentManager.findFragmentByTag(PrepublishingBottomSheetFragment.TAG)
                 if (fragment == null) {
-                    val prepublishingFragment = newInstance(site, editPostRepository.isPage)
+                    val prepublishingFragment = newInstance(
+                            site = site,
+                            isPage = editPostRepository.isPage,
+                            isStoryPost = false
+                    )
```

Comment:
can we consist these 2 flags into an enum that holds `Post, Page, Story post`  type of values? So we wouldn't be passing around 2 variables when we could pass one, and moreover we would make sure to not allow any ambiguities or impossible states such as `isPage` and `isStoryPost` being both true (which cannot be)

Reply:
This is a really important improvement. I will track this and get it done when I get some free time 🙏 https://github.com/wordpress-mobile/WordPress-Android/issues/12316